src/main/clojure/clara/rules/durability.clj

```diff
@@ -440,45 +435,27 @@
      (eng/assemble {:rulebase rulebase
                     :memory (eng/local-memory rulebase
                                               (clara.rules.engine.LocalTransport.)
-                                              (eng/options->activation-group-sort-fn opts)
-                                              (eng/options->activation-group-fn opts)
+                                              (:activation-group-sort-fn rulebase)
+                                              (:activation-group-fn rulebase)
                                               ;; TODO: Memory doesn't seem to ever need this or use
                                               ;; it.  Can we just remove it from memory?
-                                              get-alphas-fn)
+                                              (:get-alphas-fn rulebase))
```

Comment:
+1, that can be removed.  With these changes it isn't even used anymore; it is just a dead code path.